src/components/connection_handler/include/connection_handler/connection_handler_impl.h

```diff
@@ -646,7 +649,14 @@ class ConnectionHandlerImpl
    **/
   void RemoveConnection(const ConnectionHandle connection_handle);
 
-  void OnConnectionEnded(const transport_manager::ConnectionUID connection_id);
+  /**
+   * @brief Called when connection is closed.
+   *
```

Comment:
@SNiukalov please remove an empty line

Reply:
@LitvinenkoIra 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/136/commits/56d01fc91f5af95f609d1a7d3d91e73648eacb97